src/handler/events/readyHandler.ts

```diff
@@ -36,13 +39,13 @@ export function makeReadyEvent(
         .pipe(
             buildCommandModules(commandDir, sEmitter),
             callInitPlugins({
-                onFailure: module => {
+                onStop: module => {
                     sEmitter.emit(
                         'module.register',
```

Comment:
Nah, I just wanted that method to reflect what's going on with better names. Because that resolves all controllers and controller has controller.next() and controller.stop()

Reply:
wait, I meant `module.register`